### PR TITLE
Add support for the spl_token revoke instruction

### PR DESCRIPTION
**Pull request by RohanKapurDEV**
This PR adds support for the spl_token `revoke` instruction to anchor_spl

### CODE REVIEW

File: spl/src/token.rs [hunk 1]
```diff
@@ -270,6 +285,12 @@ pub struct Approve<'info> {
     pub authority: AccountInfo<'info>,
 }
 
+#[derive(Accounts)]
+pub struct Revoke<'info> {
+    pub source: AccountInfo<'info>,
+    pub owner: AccountInfo<'info>,
```
Comment (armaniferrante) [inline spl/src/token.rs:291]: Please rename `owner` to `authority`.

Comment (RohanKapurDEV) [reply spl/src/token.rs]: Done

**Comment by armaniferrante** [comment]
Please add a changelog entry.

**Comment by RohanKapurDEV** [comment]
Changelog entry added

**Comment by RohanKapurDEV** [comment]
Ignore last commit. Formatter did some things to the changelog it's not supposed to. Will fix

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,513 +11,513 @@ incremented for features.
 
 ## [Unreleased]
 
+spl: Add support for revoke instruction ([#1493](https://github.com/project-serum/anchor/pull/1493)).
+
 ### Fixes
 
-* cli: Fix rust template ([#1488](https://github.com/project-serum/anchor/pull/1488)).
```
Comment (armaniferrante) [inline CHANGELOG.md:16]: Please undo all these formatting changes.

Comment (RohanKapurDEV) [reply CHANGELOG.md:16]: Done (i think). im sorry about these silly changes. vscode hates me, apparently

**Comment by RohanKapurDEV** [comment]
Merge conflict resolved

**Review by fanatid — APPROVED** [review]

**Pull request merged**